configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fakechroot
+  - fakeroot
+  - fuse-sshfs
+  - gperftools
+  - gtk-vnc
```

Comment:
This is needed by desktop apps, so can't be in the virt module AFAIK

Reply:
We are trying to determine the list of packages that will be owned by the Virt SST, and not the virt module. 
Does `gtk-vnc` below with sst_virt ?or should it be moved to another SST ?